INSTALL.md

```diff
@@ -0,0 +1,100 @@
+# Building and Installation
+
+## Dependencies
+
+Ensure the development version of the dependencies are installed. These are as follows:
+
+* Tango Controls 9 or higher.
```

Comment:
Should we say that, if wanted, but discouraged, can be compiled against Tango 8 too ?
There are people around using HDB++ with Tango 8 actually.

Reply:
I can certainly add that for all the repositories.

 Must admit I have not tried compiling and testing with tango 8. I assume it would still work.